### PR TITLE
Clarifying implications of apache web management

### PR DESCRIPTION
**Pull request by hflynn**
See https://trello.com/c/SOvbfoLq/57-docs-pr-omero-web-pr-re-web-start-stop-gunicorn and https://github.com/openmicroscopy/openmicroscopy/pull/4618
This clarifies the doc as clearly not everyone understands the implications of the current info.

### CODE REVIEW

**Comment by hflynn** [comment]
Will be staged at http://www.openmicroscopy.org/site/support/omero5.2/sysadmins/unix/install-web/install-apache.html#running-omero-web-unix-linux for review tomorrow

**Comment by ghost** [comment]
The text looks fine.

@hflynn The URL doesn't show the change for me.  Edit: Added -staging and it looks fine.

Please merge.

**Comment by hflynn** [comment]
Thanks. Sorry about the wrong URL

File: omero/sysadmins/unix/install-web/install-apache.txt [hunk 1]
```diff
@@ -93,4 +93,6 @@ Running OMERO.web (Unix/Linux)
 
 OMERO.web is used in 'daemon' mode where UNIX sockets are used to communicate
 between the Apache child processes and the daemon processes which are
-to handle a request.
+to handle a request. **OMERO.web does not provide any management for these
+'daemon' processes;** ``bin/omero web start/status/stop``
```
Comment (atarkowska) [inline omero/sysadmins/unix/install-web/install-apache.txt:97]: I think there is one daemon process

**Pull request merged**